src/language/typescript/3.0/serializers/parameter-object.ts

```diff
@@ -121,26 +121,32 @@ const getFn = (
 	if (PrimitiveSchemaObjectCodec.is(schema)) {
 		return right(
 			serializedFragment(
-				`value => serializePrimitiveParameter('${style}', '${parameter.name}', value)`,
-				[serializedDependency('serializePrimitiveParameter', pathToUtils)],
+				`value => option.fromEither(serializePrimitiveParameter('${style}', '${parameter.name}', value))`,
+				[
+					serializedDependency('option', 'fp-ts'),
```

Comment:
although it's ok to import modules from fp-ts index in the end project, here in the lib we'd like to minimise imports, so could you import `fromEither` directly from `fp-ts/lib/Option` here?

Reply:
Aren't we going to have collisions if some other serializedFragment imports fromEither from another module?

Reply:
The same is for `option`/`either` etc from `fp-ts`. We don't have general solution at the moment.